nidirect_cold_weather_payments/src/Form/ColdWeatherPaymentCheckerForm.php

```diff
@@ -101,6 +102,9 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       ],
       '#ajax' => [
         'callback' => '::cwpCheck',
+        'wrapper' => 'cwp-results-container',
+        'effect' => 'fade',
+        'method' => 'replace',
       ],
```

Comment:
Just adding a fade effect to results which makes repeated submissions of the form more noticeable